apps/src/music/views/MusicView.jsx

```diff
@@ -237,11 +255,31 @@ class UnconnectedMusicView extends React.Component {
     this.props.setCurrentProgressState(currentState);
 
     // Tell the external system (if there is one) about the success.
-    if (this.props.levels && currentState.satisfied) {
+    if (this.hasLevels() && currentState.satisfied) {
       this.props.sendSuccessReport('music');
     }
   };
 
+  // Returns whether we just have a single level.
+  hasLevel = () => {
+    return !!this.props.currentLevelId;
+  };
+
+  // Returns whether we have levels.
+  hasLevels = () => {
+    return !!this.props.levels;
+  };
+
+  // Returns whether we have a progression file.
+  hasProgressionFile = () => {
```

Comment:
I think it's worth keeping around for now in case it helps with prototyping progressions; having it on S3 was intentional to allow various internal folks to try different ideas quickly.  But I can imagine it goes away later.